src/components/registration-lite.js

```diff
@@ -88,6 +94,7 @@ const RegistrationLite = (
         removeReservedTicket,
         reserveTicket,
         payTicketWithProvider,
+        trackEvent,
         onPurchaseComplete,
```

Comment:
lets replace on purchase complete with the new track event approach
so we could replace with analytic event like its done here
https://github.com/OpenEventKit/event-site/blob/2d9c05614901b44f9252adf67b4bd0abb83d5385/src/components/RegistrationLiteComponent.js#L163
have in consideration that we need to keep the logic of set order on event site so we will need to write a wrapper around here
https://github.com/OpenEventKit/event-site/blob/094b89ee3d4c66b0c4eadd9049cb79651dd66fb7/src/components/RegistrationLiteComponent.js#L168
in order keep calling setUserOrder like here
https://github.com/OpenEventKit/event-site/blob/094b89ee3d4c66b0c4eadd9049cb79651dd66fb7/src/components/RegistrationLiteComponent.js#L166